src/Autosuggest.js

```diff
@@ -89,6 +104,34 @@ class Autosuggest extends Component {
     }
   }
 
+  componentWillUnmount() {
+    global.window.removeEventListener('touchstart', this.onDocumentTouchStart, false);
+    global.window.removeEventListener('touchend', this.onDocumentTouchEnd, false);
+    global.window.removeEventListener('mouseup', this.onDocumentMouseUp, false);
+  }
+
+  onDocumentTouchStart({ touches }) {
+    this.touchStart = this.touchStart || touches[0];
+  }
+
+  onDocumentTouchEnd({ changedTouches }) {
+    const { clientX, clientY } = changedTouches[0];
+    const oldX = this.touchStart && this.touchStart.clientX || Number.MAX_VALUE;
+    const oldY = this.touchStart && this.touchStart.clientY || Number.MAX_VALUE;
+
+    if (!this.justTouchedInput && !this.justClickedOnSuggestion &&
+      Math.abs(oldX - clientX) < 20 && Math.abs(oldY - clientY) < 20) {
```

Comment:
My goal is to blur the input if the user taps on the page, but not blur it if the user scrolls or zooms. So I work out if the first touch object has moved more than 20px in the x or y direction since the `touchstart` event. Ideally I would bind to the `click` event, but on ios the click event is not bubbled to the document unless the target has css `cursor: pointer` http://www.quirksmode.org/blog/archives/2010/10/click_event_del_1.html